com.unity.render-pipelines.universal/Tests/Runtime/RuntimeTests.cs

```diff
@@ -64,12 +68,12 @@ public IEnumerator PipelineSetsAndRestoreGlobalShaderTagCorrectly()
 
     void AssetCheck()
     {
-        //Assert.IsNotNull(currentAsset, "Render Pipeline Asset is Null");
+        //Assert.IsNotNull(currentAssetGraphics, "Render Pipeline Asset is Null");
         // Temp fix, test passes if project isnt setup for Universal RP
-        if (currentAsset == null)
+        if (RenderPipelineManager.currentPipeline == null)
```

Comment:
`currentPipeline` returns an instance of the pipeline rather than the asset, either grab the asset file, or down further check against `UniversalRenderPipeline` rather than `UniversalRenderPipelineAsset`

Reply:
so i am unsure about what this should check then. is it that an URP asset will be selected to be the active pipeline? (if so it means a combination of graphics/quality SRP asset?

Reply:
Oh actually it's not so much an issue with this line, more that `currentAsset` was switched out for `RenderPipelineManager.currentPipeline`, the real issue is actually a few lines lower where it does the `AreEquals` check

Reply:
oh yep! okay will rerun locally on batch mode and editor mode and compare everything is green!